docs/en/developers/06_Reporting_Bugs.md

```diff
@@ -64,3 +66,17 @@ Remember to give the following information if you know it:
 3. Which version of PHP?
 4. Which database: SQLite, MySQL, MariaDB, PostgreSQL? Which version?
 5. Which distribution runs on the server? And… which version?
+
+## How to provide feed data
+
+If you need us to investigate a feed problem, it will be easier if you provide a snapshot of the feed data.
+To do that, you can launch the following command:
+
+```bash
+wget <feed url> -O output.rss
+```
+As github does not allow to attach files to an issue, it’s better to create a public gist containing the file.
```

Comment:
But it is possible to attach files to an issue ;-)

Reply:
In fact it's preferable because the chance of it working a few months or sometimes even a few days from now is significantly higher if it's attached to GH.

But for a file like output.rss you'll either need to rename it to output.rss.txt or zip it. (Or rename it to output.rss.zip but that would be lacking in clarity. ^_^)

Reply:
Pardon, I'd overlooked the gist part which is also part of GH but what I said is still true. Gists disappear if users do some cleanup or if their account is deleted, and I'm pretty sure there are at least two distinct issues in this very repo where a gist has disappeared, while an attached (zipped) file would still be there.

Reply:
I couldn't find a way to do that but I agree with you. It's better to keep it in the issue. Could you tell me where I can find info on that so I can change that part?

Reply:
Thank you!

Reply:
I've made the modification.